### DISCUSSION TITLE
Led activation based on external software

### DISCUSSION THREAD

**Discussion by qchaillet** [shanteacontrols/OpenDeck#114] [General]
Hello,

I was wondering if I can toggle a led on and off based on when the effect is active on my daw, 

(that effect is activable with CC )

Thank you :-)

**Comment by paradajz** [comment]
Yes. Check this page for more info:

https://github.com/shanteacontrols/OpenDeck/wiki/LED-control

**Reply by qchaillet** [reply]
Great ! Sorry I read the wiki before but my lack of midi knowledge made it hard to understand, thanks I will give it a try !
